scripts/prepare/bundle.ts

```diff
@@ -117,11 +120,24 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
         platform: platform || 'browser',
         external: externals,
 
+        esbuildPlugins: [
+          esbuildAliasPlugin({
+            lodash: 'lodash-es',
```

Comment:
> we have to also make sure, that lodash-es is installed for every package

yes

> Does the aliasing also encounter for lodash usage in node_modules?

Sort of, when the bundler is following the modules-graph up, it will encounter imports to `lodash` and swap it out.
However when it encounters a module that's external, it will skip, and leave the import as-is.

This means if `package-a` depends on `lodash`, but `package-a` is externalized, it will NOT be swapped.